app/src/main/res/values/styles.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <style name="SplashTheme" parent="Theme.AppCompat.NoActionBar">
```

Comment:
this should be called SplashStyle instead of SplashTheme as it is not a theme

Reply:
or put it into a theme xml file

Reply:
done